com.maxizamorano.photopea-no-ads/com.maxizamorano.photopea-no-ads.yml

```diff
@@ -0,0 +1,41 @@
+app-id: com.maxizamorano.photopea-no-ads
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+command: photopea_no_ads.sh
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-download
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: photopea_no_ads
+    buildsystem: simple
+    build-options:
+        env:
+          XDG_CACHE_HOME: /run/build/photopea/cache
+    build-commands:
+      - install -D -t /app/bin photopea_no_ads.sh
+      - mkdir /app/main
+      - tar -xzf photopea_no_ads_build.tar.gz -C /app/main
+      - install -Dm755 /app/main/resources/data/com.maxizamorano.photopea-no-ads.desktop -t /app/share/applications
+      - install -Dm644 /app/main/resources/data/com.maxizamorano.photopea-no-ads.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 /app/main/resources/icons/photopea.png /app/share/icons/hicolor/512x512/apps/com.maxizamorano.photopea-no-ads.png
+    sources:
+      - type: file
+        url: https://github.com/MaxiZamorano/photopea-no-ads/releases/download/v1.0.0/photopea_no_ads_build.tar.gz
```

Comment:
how does that work on aarch64?

Reply:
This should be built from source.

Reply:
> ¿Cómo funciona eso en Aarch64?

I haven't tested it on aarch64, only x86_64.

Reply:
but an aarch64 package is still built with these x86_64 binaries. Why not building from source?